Libplanet.Tools/FileConfigurationService.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using Zio;
+
+namespace Libplanet.Tools
+{
+    public class FileConfigurationService : IConfigurationService<string, string>
+    {
+        private readonly IFileSystem _fileSystem;
+
+        public FileConfigurationService(IFileSystem fileSystem)
+        {
+            _fileSystem = fileSystem;
+        }
+
+        public string Get(string key)
+        {
+            if (!_fileSystem.FileExists(KeyPath(key)))
+            {
+                throw new KeyNotFoundException($"The given key, '{key}' does not exist.");
+            }
+
+            return _fileSystem.ReadAllText(KeyPath(key));
```

Comment:
Rather than implementing key-value configuration using multiple files (where filenames are keys), IMHO it's more handy to maintain a single JSON/YAML/TOML file.

It would be okay with any format at this moment, because we can change the format before releasing v0.10.0.  So the easiest one to implement it.  Or leave a `FIXME` comment and file an issue addressing this and let's move fast.

Reply:
I tried to find ways to stay it now interface and to maintain with JSON. But it was hard to use strict type because they must be `Dictionary<string, object>` type. So I introduced a new `ToolConfiguration` struct class and changed the `IConfigurationService` interface. Now the interface provides methods only `Load` and `Store`. Another instruction can proceed like handling a struct variable, with strict type 😀